README.md

```diff
@@ -1,61 +1,83 @@
+    <script src="https://unpkg.com/@accept-nano/client"></script>
+  </head>
+  ...
 </html>
 ```
 
-## Usage
+## Using
 
-After the _accept-nano-client_ script is loaded there will be a global variable called _acceptNano_, which is an instance of our primary AcceptNano class and will be used for communicating your server, starting and controlling payment sessions.
+### Creating a Payment Session
 
-### Setting up the Client
+To be able to initiate the payment process, you **must create a new payment session.**
 
-Before initiating the payment process, you **must** configure the _acceptNano_ instance by using the following method and parameters:
-
-```JS
-window.acceptNano.setup({
-    apiURL: 'api.myAcceptNanoServer.com', // URL of your Accept NANO server (String, required)
-    debug: false, // used for enabling debug mode, (Bool, non-required, false by default)
-    pollInterval: 1500, // time period (ms) to check for verification of the payment sessions (Number, non-required, 1500 by default)
+```ts
+const session = acceptNano.createSession({
+  apiURL: 'api.myAcceptNanoServer.com',
+  pollInterval: 1500,
 })
+
+// register your event listeners to shape-up the logic based on session events.
+session.once('start', () => { ... })
+session.once('success', (payment: AcceptNanoPayment) => { ... })
+session.once('failure', (reason: AcceptNanoPaymentFailureReason) => { ... })
 ```
 
-### Initiating Payment Session
-
-After you configure the acceptNano instance, you can initiate a payment session by using the following:
-
-```JS
-window.acceptNano.startPayment({
-    data: {
-        amount: '10' // (String, required)
-        currency: 'USD' // (String, required, 'USD' or 'nano')
-        state: '{userId:7}' // State to share between client and server, (String, non-required)
-    },
-    onStart: (paymentData) => {} // Function, fired when the payment session starts
-    onSuccess: (paymentData) => {} // Function, fired when the payment session completed succesfully
-    onFailure: (failureReason) => {} // Function, fired when the payment session fails
-    onCancel: () => {} // Function, fired when the payment session is cancelled by the user
+### Presenting the Payment Overlay
+
+After creating your session and attaching the event listeners, you can follow one of those options to proceed with the payment flow.
+
+#### Option 1: Create a Payment Through Client
+
+If you want to create and verify an _accept-nano_ payment in your client application, you can use this option.
+
+```ts
+session.createPayment({
+  amount: '1',
+  currenct: 'USD',
+  state: '{userId:7}',
 })
 ```
 
+After the payment is created, the client will automatically proceed to the verification step.
+
+#### Option 2: Verify a Payment Through Client
+
+If you create an _accept-nano_ payment in another context (such as your application's backend), you can use this option to perform the verification on client.
+
+```ts
+session.verifyPayment('eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9')
```

Comment:
Maybe we should tell that the argument is the token from payment creation endpoint.

Reply:
https://github.com/accept-nano/accept-nano-client/commit/5dd0679830374542796efb51ff25759f4bb14ac1 done 👍